.kitchen.yml

```diff
@@ -16,7 +16,8 @@ platforms:
   - name: debian-8.4
   - name: windows
     driver:
-      box: opentable/win-2012r2-standard-amd64-nocm
+      box: mwrock/Windows2012r2
+      gui: false # This box apparently defaults to using a GUI.
```

Comment:
Thanks @mwrock! I'm likely just an odd bear then, tending to prefer headless. But if the masses have spoken. 👍 

(nit-picky comment for @alecxvs then: The comment on this line should be clarified to reflect this fact maybe something like `# Most users of this box want the GUI, therefore it defaults to having one that we don't need.` Knowing myself, I am liable to comment on it again in a future review forgetting that we know the reasoning behind the box setting and we're not just going "well it appears this way" )